src/runtime/utils/wixStyleFontUtils.ts

```diff
@@ -63,17 +63,20 @@ export const wixStylesFontUtils = {
 
     return ret;
   },
-  toFontCssValue(value) {
+  toFontCssValue(value: Required<IStyleFont>) {
     const size = isNumber(value.size) ? value.size + 'px' : value.size;
     const lineHeight = isNumber(value.lineHeight) ? value.lineHeight + 'px' : value.lineHeight;
-    const family = value.family.map(val => (val.indexOf(' ') > -1 ? JSON.stringify(val) : val)).join(',');
+    const family =
+      typeof value.family === 'string'
+        ? value.family
+        : value.family.map((val: string) => (val.indexOf(' ') > -1 ? JSON.stringify(val) : val)).join(',');
```

Comment:
found that `family` could be defined as `string` and as an `array of strings`, so tuned this logic a bit.

**TBD**